src/register/Register.jsx

```diff
@@ -68,8 +68,9 @@ export default function Register() {
                     <form className="register-form mx-auto" onSubmit={handleSubmit}>
                         <form-group controlId="formName">
                             <p className="input-control">
-                                <label htmlFor="name">Name :</label>
-                                <input className="field"
+                                <label id="Name">Name :</label>
+                                <input 
+                                    className="field"
                                     type="text"
                                     name="name"
```

Comment:
Add aria-labelledby attribute like how it is done in the `PersonalDetails.jsx`
<img width="583" alt="Screen Shot 2020-09-03 at 7 03 37 pm" src="https://user-images.githubusercontent.com/29667122/92095230-badea280-ee18-11ea-8888-8974b743ad4a.png">

as it is pointed out by the warning
<img width="719" alt="Screen Shot 2020-09-03 at 7 08 07 pm" src="https://user-images.githubusercontent.com/29667122/92095299-d2b62680-ee18-11ea-8c08-e9ee8c491891.png">

Do this to all other input fields where `required` attribute is used.